doc/rst/language/spec/variables.rst

```diff
@@ -132,11 +132,11 @@ omitted, the compiler will search forward in the function for the
 earliest assignment statement(s) setting that variable that occur before
 the variable is otherwise mentioned. It will consider the variable passed
 to an ``out`` intent argument as an assignment statement for this
-purpose.  It will search only within block declarations ``{ }``, ``try``
-blocks, ``try!`` blocks, and conditionals.  These assignment statements
-and calls to functions with ``out`` intent are called applicable
-assignment statements.  They perform initialization, not assignment, of
-that variable.
+purpose.  It will search only within block declarations ``{ }``,
```

Comment:
This is a pre-existing condition, but I'd call ``{ .. }`` a "compound statement" or possibly "block statement" or maybe "compound (block) statement" (which ties it in to the following "block" forms a bit) rather than a "block declaration".

Earlier in this paragraph:
* "search forward in the function" — it's not necessarily a function, is it?  (could be a module, could be a compound statement).  What about "search forward in the scope"?
* probably "occur" should become "occurs" to match the (default) "statement" or else "occur(s)" to match "statement(s)" (yet with reverse polarity)?  though maybe there's a grammar/style rule here that I'm not aware of (my intuition is that since the "(s)" is optional, it shouldn't influence the choice of the following word by default).

Reply:
I'll go with "block statement" to match https://chapel-lang.org/docs/master/language/spec/statements.html#blocks .
I'll make the other changes.